api/api/services/internal_tasks/internal_tasks_service.py

```diff
@@ -731,8 +741,13 @@ async def _fetch_previous_task_inputs(
         # Ultimately we should integrate the fetching to the SDK. `run_task_input_example_task.list_runs`
         # To ensure that we always fetch from the same environment.
 
+        task_input_example_task_uid = await self._get_task_input_example_task_uid()
+        if not task_input_example_task_uid:
+            self.logger.exception("Can't find the task input example task")
+            return None
+
         previous_run_query = SerializableTaskRunQuery(
```

Comment:
TASK_INPUT_EXAMPLE_TASK_ID is not that useful anymore not that I use the agent_uid, but that can't hurt.